xml/System.Collections.Immutable/ImmutableDictionary`2.xml

```diff
@@ -1704,9 +1705,9 @@
         <Parameter Name="items" Type="System.Collections.Generic.IEnumerable&lt;System.Collections.Generic.KeyValuePair&lt;TKey,TValue&gt;&gt;" />
       </Parameters>
       <Docs>
-        <param name="items">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="items">The key=value pairs to set on the map. Any keys that conflict with existing keys will overwrite the previous values.</param>
```

Comment:
I'd personally use "replace" or "update" instead of "overwrite", as the latter more strongly implies mutation.

Reply:
```suggestion
        <param name="items">The key-value pairs to set on the map. Any keys that conflict with existing keys will replace the previous values.</param>
```